<launch>
	<node pkg="restart" type="marker_test" name="marker_test" output="screen"/>
	<node pkg="vicon_bridge" type="vicon_bridge" name="vicon" >
	    <param name="stream_mode" value="ServerPush" type="str" />
	    <param name="datastream_hostport" value="192.168.2.100:801" type="str" />
	    <!--param name="datastream_hostport" value="vicon:801" type="str" /-->
	    <param name="tf_ref_frame_id" value="/world" type="str" />
	</node>
</launch>